aws/v9.3.4/release.yaml

```
@@ -0,0 +1,50 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v9.3.4
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - name: chart-operator
+    version: 0.13.0
+  - componentVersion: 1.16.2
+    name: cluster-autoscaler
+    version: 1.16.0
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.1.10
+  - componentVersion: 1.9.5
+    name: kube-state-metrics
+    version: 1.1.0
+  - componentVersion: 0.3.3
+    name: metrics-server
+    version: 1.1.0
+  - name: net-exporter
+    version: 1.9.0
+  - componentVersion: 0.34.0
+    name: nginx-ingress-controller
+    version: 1.7.2
+  - componentVersion: 0.18.1
+    name: node-exporter
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-operator
+    version: 5.7.1
+  - name: cert-operator
+    version: 0.1.0
+  - name: cluster-operator
+    version: 0.23.10
```

```suggestion
    version: 0.23.12
```
TODO once it's released